host_vars/klunker-ap3/base.yml

```diff
@@ -0,0 +1,8 @@
+---
+
+location: klunker
+role: ap
+model: "mikrotik_sxtsq-5-ac"
+mac_override:
+  eth0: 08:55:31:14:36:d7
+wireless_profile: freifunk_default
```

Comment:
this line is unncessary, but not harmful. freifunk_default is default for an ap (but not the core)